kyuubi-hive-jdbc-shaded/pom.xml

```diff
@@ -112,6 +112,19 @@
             </exclusions>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.hive</groupId>
+            <artifactId>hive-standalone-metastore</artifactId>
```

Comment:
r u missing `hive-llap-client`?

Reply:
we should udpate LICENSE and NOTICE file under the shaded module when change the shaded content.

Reply:
> r u missing `hive-llap-client`?

hive-llap-client is in kyuubi-hive-jdbc/pom.xml

Reply:
> we should udpate LICENSE and NOTICE file under the shaded module when change the shaded content.

done